languages/English

```diff
@@ -0,0 +1,134 @@
+69=buy extra lives, 4000 points per life
+70=Buy potions, 1000 points per potion
+71=buy ammo, 900 points for each pack of 30 ammo
+72=buy booby traps, 1200 points for each set of 5 booby traps
+73=buy armor, 800 points for each armor
+74=buy evade blades items, 1500 points per item
+75=Welcome to the battle zone shop! You have 
+76= points that you can give me! Select an item you wish to buy!
+77=ammo packs
+78=booby trap sets
+79=armor ratings
+80=How much of the item do you want to buy?
+81=so. You want to buy 
+82=. I hope you have 
+83= points to give me, because that's how much it will cost. You still want to buy it?
+84=What do you think you are doing? Are you trying to cheat the shopkeeper? Well, you don't have enough points to buy stuff. Sorry man, come back later, and don't cheap me next time!
+85=oh wait, you're playing in reality mode. I'm sorry, but we're sold out on extra lives. Here's your money bac.
+86=you don't have any more evade blade items!
+87=boring blast
+88=Medium mass murder
+89=Advanced arson
+90=I just wanna die now!
+91=Choose your difficulty level!
+92=cheats for Battle zone!	To activate a cheat, just type the command into the game. They work at most times.	Hold down delete and press z: give you 250 armor	Alt+backslash: evade blades forever	Control+f9: skip this level and go on to the next	hold down space and press 9: give you a whole whole whole whole whole lot of ammo!	Hold down f4 and press enter: give you 250 extra lives!	Hold down enter and press x: give you 500 more potions!	Hold down u, then hold down space, and then press left shift: give you 500 extra booby traps!
+93=For gaining over 20000 points in the game, you have just received a cheats file! It can be found in your battle zone directory.
+94=Getting out of the lab
+95=scenes from country, I mean city, life
+96=The water of death
+97=Street know how
+98=Forest excursions
+99=rumbles in the dark
+100=A winter wonder land, full of enemies ...
+101=The Vile River
+102=Cave of violence
+103=Danger on the road
+104=caught in the storm!
+105=Your game has been saved!
+106=It seems that the save file was not found. Save a game and try again.
+107=options were saved!
+108=The boss is dead! You have now gained 7000 points!
+109=Your total score is: 
+110=Item
+111=item falls
+112=You collected ammo
+113=You collected potion
+114=You collected armor
+115=you collected evade blades item
+116=Ahwhwhwhwhwhwhwhwhwh, you got a fake item, too bad!
+117=Pit
+118=Enemy jumps over a pit
+119=enemy falls into a pit
+120=booby trap
+121=Enemy accidentally stepped onto a booby trap and just died!
+122=you accidentally step on to your own booby trap and go boom!
+123=blades
+124=blades cut you
+125=blades protection destroyed
+126=boulder
+127=boulder hits enemy
+128=boulder hits you
+129=Enemy attack completely absorbed by armor
+130=Fireball
+131=Fireball hits you
+132=you are injured
+133=end of level
+134=Use up and down arrow keys to navigate, and press enter to select.
```

Comment:
Still, what's this "no newline at end of file" thing?

Reply:
No new line at end of file? I'm sorry, I don't know what you mean.